src/http/websocket_http_client.zig

```diff
@@ -1596,8 +1624,10 @@ pub fn NewWebSocketClient(comptime ssl: bool) type {
                 this.clearData();
                 return;
             }
-
-            socket.shutdownRead();
+            // we dont wanna shutdownRead when SSL, because SSL handshake can happen when writting
+            if (comptime !ssl) {
```

Comment:
Is there an OpenSSL version of shutdown read? One which doesn’t set it on the socket but tells ssl that no more non-handshaking data

Reply:
not aware of it, will read some documentation and fix this internally on usockets so we dont need to worry about this on Zig side

Reply:
OpenSSL can only shutdown both write and read if we close one will sent/receive `close_notify` so we should not call `shutdownRead` or any form of it for TLS if we wanna to write more data after it.